spec/game_runner_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'spec_helper.rb'
+require_relative '../lib/game_runner.rb'
+
+RSpec.describe Game, "#run" do
+  context "on start up" do
+    it "renders an empty grid on the screen" do
+      game = Game.new
+      horizontal_line = "  __ __ __"
+      vertical_line = " |  |  |  |"
+      both_lines = ' |__|__|__|'
+      new_line = "\n"
+
+      game.run
+
+      expect(output).to eq((horizontal_line + new_line + vertical_line + new_line + both_lines + new_line)*3)
```

Comment:
`output` doesn't exist. So it's really unclear what is happening here. 

When you run the tests, what is the error? A good failing test would say, expected this thing to be __SOME STRING THAT IS THE GRID __ and it is empty instead. I think the test failure is going to be unknown variable output. 

So, the test isn't properly failing.

Reply:
Got it. What do you think of the updated?